src/assets/i18n/es.json

```diff
@@ -90,11 +90,11 @@
 		"languages": {
 			"cnil_translations": "Traducciones del CNIL",
 			"community_translations": "Traducciones de la comunidad",
-			"cz": "Česky",
+			"cz": "Cesky",
```

Comment:
Please keep the original traduction.

Reply:
Ok

Reply:
Could you keep the original traduction? It must be "Česky". thx